src/Pages/Dashboard.js

```diff
@@ -11,45 +11,75 @@ class Dashboard extends Component {
 
     this.auth = new AuthService()
     this.state = {
-      tasks: [],
+      myTasks: [],
     }
   }
 
-  addDays = function(days) {
-    // task = this.props.due_date value
-    let date = new Date()
-    date.setDate(date.getDate() + days);
-    return date;
-  }
-
-  render() {
-    if (true) {
+  render = () => {
+    let { myTasks } = this.props
+    let today = new Date()
```

Comment:
I've noticed that as a team you use `let` by default.  `let` should be used for variables that will change.  If it's a value like `today`, which should not change, then use `const`.   

@l1ngoman @jscotty723 @kristenmccloud @ChristopherJStubbs